CRM/Utils/PagerAToZ.php

```diff
@@ -151,7 +151,7 @@ public static function createLinks(&$query, $sortByCharacter, $isDAO) {
       $qfKey = CRM_Utils_Array::value('qfKey', $query->_formValues);
     }
     if (empty($qfKey)) {
-      $qfKey = CRM_Utils_Request::retrieve('qfKey', 'String', $this, FALSE, NULL, $_REQUEST);
+      $qfKey = CRM_Utils_Request::retrieve('qfKey', 'String', NULL, FALSE, NULL, $_REQUEST);
```

Comment:
@eileenmcnaughton @seamuslee001 I just tested it and got the same error.

      $empty = NULL;
      $qfKey = CRM_Utils_Request::retrieve('qfKey', 'String', $empty, FALSE, NULL, $_REQUEST);
seems to work, as we are working with a variable and not a constant.